Jenkinsfile

```diff
@@ -181,6 +181,25 @@ pipeline {
           }
         }
 
+        stage('JDK 11 and Scala 3.0') {
```

Comment:
We wouldn't want to have a 4th PR build, we would switch one of the existing ones to use Scala 3.0.

Reply:
Makes sense. Jenkins doesn't pick up the configuration from the PR changes themselves right? On the tasks associated with the PR I didn't see any Scala 3.0 pipeline.

Reply:
For security reasons, it won't do it until it's merged if the submitter is not a committer.

Reply:
Makes complete sense. I was assuming this was the behavior.
I wait to change this once the PR is really actionable.